hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/RenameAtomicityUtils.java

```diff
@@ -286,58 +268,13 @@ private String makeRenamePendingFileContents(List<BlobProperty> blobPropertyList
     sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
     String time = sdf.format(new Date());
 
-    // Make file list string
-    StringBuilder builder = new StringBuilder();
-    builder.append("[\n");
-    for (int i = 0; i != blobPropertyList.size(); i++) {
-      if (i > 0) {
-        builder.append(",\n");
-      }
-      builder.append("    ");
-      final String noPrefix;
-      /*
-       * The marker file for the source directory has the same path on non-HNS.
-       * For the other files, the fileList has to save the filePath relative to the
-       * source directory.
-       */
-      if (!blobPropertyList.get(i)
-          .getPath()
-          .toUri()
-          .getPath()
-          .equals(srcPath.toUri().getPath())) {
-        noPrefix = StringUtils.removeStart(
-            blobPropertyList.get(i).getPath().toUri().getPath(),
-            srcPath.toUri().getPath() + FORWARD_SLASH);
-      } else {
-        noPrefix = "";
-      }
-
-      // Quote string file names, escaping any possible " characters or other
-      // necessary characters in the name.
-      builder.append(quote(noPrefix));
-      if (builder.length() >=
-          MAX_RENAME_PENDING_FILE_SIZE - FORMATTING_BUFFER) {
-
-        // Give up now to avoid using too much memory.
-        LOG.error(
-            "Internal error: Exceeded maximum rename pending file size of {} bytes.",
-            MAX_RENAME_PENDING_FILE_SIZE);
-
-        // return some bad JSON with an error message to make it human readable
-        return "exceeded maximum rename pending file size";
-      }
-    }
-    builder.append("\n  ]");
-    String fileList = builder.toString();
-
     // Make file contents as a string. Again, quote file names, escaping
```

Comment:
Why are removing this code piece ?

Reply:
We are not keeping fileList. The json will just have src and dst.